### PR TITLE
NN-3625 changing the name in the helm charts and circle ci config pre…

### PR DESCRIPTION
**Pull request by richardpopple**
…mptively

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@
 replicaCount: 4
 
 image:
-  repository: quay.io/hmpps/prisonstaffhub
+  repository: quay.io/hmpps/digital-prison-services
   tag: latest
```
Comment (richardpopple) [inline helm_deploy/values-preprod.yaml:8]: I have not changed the name of the cert_secret2 property in this file
cert_secret2: prison-staff-hub-cert
As I assume that this is pulled from somewhere with this name

File: helm_deploy/values-dev.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@
 replicaCount: 2
 
 image:
-  repository: quay.io/hmpps/prisonstaffhub
+  repository: quay.io/hmpps/digital-prison-services
   tag: latest
```
Comment (richardpopple) [inline helm_deploy/values-dev.yaml:8]: I have not changed the name of the cert_secret2 property in this file
cert_secret2: prison-staff-hub-cert
As I assume that this is pulled from somewhere with this name

File: helm_deploy/values-prod.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@
 replicaCount: 4
 
 image:
-  repository: quay.io/hmpps/prisonstaffhub
+  repository: quay.io/hmpps/digital-prison-services
   tag: latest
```
Comment (richardpopple) [inline helm_deploy/values-prod.yaml:8]: I have not changed the name of the cert_secret2 property in this file
cert_secret2: prison-staff-hub-cert
As I assume that this is pulled from somewhere with this name

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (mattops) [reply helm_deploy/values-preprod.yaml:8]: yes the cert is created in the cloudplatform environment repo https://github.com/ministryofjustice/cloud-platform-environments/blob/main/namespaces/live-1.cloud-platform.service.justice.gov.uk/digital-prison-services-dev/07-certificates.yaml

File: helm_deploy/values-dev.yaml (hunk 2 above)
Comment (mattops) [reply helm_deploy/values-dev.yaml:8]: see other comment for above.

File: helm_deploy/values-prod.yaml (hunk 3 above)
Comment (mattops) [reply helm_deploy/values-prod.yaml:8]: see other comment above.

File: helm_deploy/values-preprod.yaml (hunk 1 above)
Comment (richardpopple) [reply helm_deploy/values-preprod.yaml:8]: I've now changed this as I think we need host2 still - I'll raise a pull request on cloud-platform-environments

**Review by mattops — APPROVED** [review]

**Pull request merged**